.travis.yml

```diff
@@ -5,10 +5,10 @@ python:
   - "3.8"
 install:
   - pip install --upgrade pip
-  - pip install numpy jaxlib tensorflow tensorflow-datasets --upgrade
+  - pip install pygame==2.0.0.dev6 tfp-nightly tfds-nightly numpy jaxlib jax tf-nightly matplotlib more-itertools --upgrade
```

Comment:
Just to double-check, are all the packages here needed? I don't see them used yet, and not sure if we need the nightly build either. Lmk!

Reply:
Yeah, a little bit of story here. Actually the above is the combination that I have figured out after many Travis CI build errors, as you can see - https://github.com/DarrenZhang01/Neural_Tangents_TensorFlow/commits/master/.travis.yml

By the way, is there anything I can do about the `jax.core` error in the latest commit?

Reply:
Re `jax.core` - make sure to remove `jax` from the install list in line 8, it is copied from github head below (because jax on pypi is not updated frequently enough)

Reply:
Got it! Thanks!

Reply:
And sure, feel free to import what you need, but If the current PR can pass tests without modifying the `.travis.yml` (I think it should be able to, or no?), then I'd postpone editing this file so that it's more clear what needs what additional packages at what point.

Reply:
OK, I see what you mean. I will postpone doing the installation changes after replacing the APIs in the core files of Neural Tangents. But we still need a place to put `lax` and `ostax`, maybe directly under tests folder?

Reply:
Please see my reply above - I believe they can be in `tf_helpers` and called from `[tf_]lax_test` and `[tf_]ostax_test` files in `tests` or `tf_tests`, but I may be wrong.

Reply:
> Please see my reply above - I believe they can be in `tf_helpers` and called from `[tf_]lax_test` and `[tf_]ostax_test` files in `tests` or `tf_tests`, but I may be wrong.

Yes, Roman, your strategy works! - https://travis-ci.org/github/google/neural-tangents/builds/721174208